pywinauto/linux/application.py

```diff
@@ -0,0 +1,171 @@
+		child_window(best_match=...) search as fallback
+        """
+        self.process = None
+        self.xmlpath = ''
+
+        self._proc_descriptor = None
+        self.match_history = []
+        self.use_history = False
+        self.actions = None # TODO Action logger for linux
+        if backend not in registry.backends:
+            raise ValueError('Backend "{0}" is not registered!'.format(backend))
+        self.backend = registry.backends[backend]
+        self.allow_magic_lookup = allow_magic_lookup
+
+    def start(self, cmd_line, timeout=None, retry_interval=None,
+              create_new_console=False, wait_for_idle=True, work_dir=None):
+        command_line = shlex.split(cmd_line)
+        try:
+            process = subprocess.Popen(command_line, shell=create_new_console)
+        except Exception as exc:
+            # if it failed for some reason
+            message = ('Could not create the process "%s"\n'
+                       'Error returned by CreateProcess: %s') % (cmd_line, str(exc))
+            raise AppStartError(message)
+        self._proc_descriptor = process
+        self.process = process.pid
+        return self
+
+    def connect(self, **kwargs):
+        """Connect to an already running process
+
+        The action is performed according to only one of parameters
+
+        :param process: a process ID of the target
+        :param path: a path used to launch the target
+
+        .. seealso::
+
+           :func:`pywinauto.findwindows.find_elements` - the keyword arguments that
+           are also can be used instead of **process**, **handle** or **path**
+        """
+        connected = False
+        if 'process' in kwargs:
+            self.process = kwargs['process']
+            assert_valid_process(self.process)
+            connected = True
+        elif 'path' in kwargs:
+            for proc_id in os.listdir('/proc'):
+                if proc_id == 'curproc':
+                    continue
+                try:
+                    with open('/proc/{}/cmdline'.format(proc_id), mode='rb') as fd:
+                        content = fd.read().decode().split('\x00')
+                except IOError:
+                    continue
+
+                if kwargs['path'] in " ".join(content):
+                    self.process = int(proc_id)
+                    connected = True
+                    break
+
+        if not connected:
+            raise RuntimeError(
+                "You must specify one of process, handle or path")
+
+    def cpu_usage(self, interval=None):
+        """Return CPU usage percent during specified number of seconds"""
+        if not self.process:
+            raise AppNotConnected("Please use start or connect before trying "
+                                  "anything else")
+        if interval:
+            time.sleep(interval)
+        try:
+            proc_info = subprocess.check_output(["ps", "-p", str(self.process), "-o", "%cpu"], universal_newlines=True)
```

Comment:
I think `%cputime` is more relevant in case of non-None `interval` value. But we have to start `ps` command twice then.